nodebuilder/node.go

```diff
@@ -149,7 +161,11 @@ func (n *Node) Stop(ctx context.Context) error {
 func newNode(opts ...fx.Option) (*Node, error) {
 	node := new(Node)
 	app := fx.New(
-		fx.NopLogger,
+		fx.WithLogger(func() fxevent.Logger {
+			zl := &fxevent.ZapLogger{Logger: fxLog.Desugar()}
+			zl.UseLogLevel(zapcore.DebugLevel)
+			return zl
+		}),
```

Comment:
It is unclear from the PR, what component would take  `fxevent.Logger` as dependancy?  Also would you consider fx log level configurable?

Reply:
This is a special WithLogger option and it's not a Provide. The given logger is only used by fx itself and not by us. 

We can now control the log verbosity of fx with this change by `fx`.

Reply:
Does this change mean that fx logs will now show up when DEBUG logs are turned on?

Reply:
Yes